samples/04-file-transfer/consumer/build.gradle.kts

```diff
@@ -26,8 +26,7 @@ dependencies {
 
     implementation(project(":extensions:in-memory:transfer-store-memory"))
     implementation(project(":extensions:in-memory:policy-registry-memory"))
-    implementation(project(":extensions:in-memory:metadata-memory"))
-    
+
```

Comment:
The dependency will probably need to be replaced with the in-memory `AssetIndex`, right?